src/translator.js

```diff
@@ -89,9 +89,64 @@ function initTurndownService() {
 	return turndownService;
 }
 
+function getHighestHeader(content) {
+	if(content.includes('<h1>')) {
+		return 1;
+	}
+	if(content.includes('<h2>')) {
+		return 2;
+	}
+	if(content.includes('<h3>')) {
+		return 3;
+	}
+	if(content.includes('<h4>')) {
+		return 4;
+	}
+	if(content.includes('<h5>')) {
+		return 5;
+	}
+	if(content.includes('<h6>')) {
+		return 6;
+	}
+	return 1;
+}
+
 function getPostContent(post, turndownService, config) {
 	let content = post.encoded[0];
 
+	content = content.replace(/<h(\d).{0,}id="(.*?)".{0,}?>(.*?)<\/h\d>/, '<h$1>anchorLink123 name="$2">/anchorLink123>$3</h$1>');
+
+	let highestHeader = getHighestHeader(content);
+	switch(highestHeader) { 
+		case 1: {
+			content = content.replaceAll('<h5>', '<h6>');
+			content = content.replaceAll('</h5>', '</h6>');
```

Comment:
This looks wrong initially, but then I realize that HTML only has 6 levels of headers. Let's maybe add a comment?